core/src/main/java/com/transferwise/common/gracefulshutdown/DefaultGracefulShutdownStrategiesRegistry.java

```diff
@@ -1,30 +1,17 @@
 package com.transferwise.common.gracefulshutdown;
 
-import java.util.ArrayList;
 import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.context.ApplicationContext;
-import org.springframework.core.annotation.AnnotationAwareOrderComparator;
 
 @Slf4j
 public class DefaultGracefulShutdownStrategiesRegistry implements GracefulShutdownStrategiesRegistry {
 
   @Autowired
-  private ApplicationContext applicationContext;
-
+  @Getter
+  @Setter
   private List<GracefulShutdownStrategy> strategies;
 
-  @Override
-  public List<GracefulShutdownStrategy> getStrategies() {
-    if (strategies == null) {
-      List<GracefulShutdownStrategy> strategies = new ArrayList<>(applicationContext.getBeansOfType(GracefulShutdownStrategy.class).values());
-
-      AnnotationAwareOrderComparator.sort(strategies);
```

Comment:
Do we not care about order anymore?

Reply:
I mean the @Autowired is already doing the ordering itself.

Reply:
Though we use `AnnotationAwareOrderComparator.sort` in one other place, when strategies have changed between initialization and startup.